automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -263,9 +263,10 @@ def product_upgrade(
         sys.exit(1)
     if not os.environ.get('SATELLITE_HOSTNAME'):
         if not sat_image and not os.environ.get('RHEV_SATELLITE'):
-            print('Please provide either Satellite RHEVM template name or '
-                  'Satellite HostName to perform upgrade!')
-            sys.exit(1)
+            sat_image = os.environ.get('SAT_IMAGE')
+            sat_host = os.environ.get('SAT_HOST')
```

Comment:
Should be checked after defining sat_image and sat_host if one of them is None and exit. That will help on setting up a new environment and will prevent further failures.